mail/templates/welcome_course_run_enrollment/body.html

```diff
@@ -0,0 +1,145 @@
+{% extends "email_base.html" %}
+
+{% block content %}
+<!-- 1 Column Text + Button : BEGIN -->
+<tr>
+  <td style="background-color: #ffffff">
+    <table
+      role="presentation"
+      cellspacing="0"
+      cellpadding="0"
+      border="0"
+      width="100%"
+    >
+      <tr>
+        <td
+          style="
+            padding: 20px;
+            font-family: sans-serif;
+            font-size: 15px;
+            line-height: 20px;
+            color: #555555;
+          "
+        >
+          <p style="margin: 0 0 10px">
+            Dear
+            {{ user.name }},
+          </p>
+          <p style="margin: 0 0 10px">Welcome to MIT xPRO!</p>
```

Comment:
According to the document provided, there are spaces before this line
```suggestion
          <p style="margin: 0 70 10px">Welcome to MIT xPRO!</p>
```

Reply:
I have set the left margin to 50 for this line as it closely matches what is in the document.
<img width="751" alt="image" src="https://github.com/mitodl/mitxpro/assets/41048311/dc1bc968-9873-4e3d-a22a-690c6960c63a">